Update for DnD5e 4.2.2 Final

Szerina92 opened the pull request:
It took a while...
Unfortunately, I haven't been able to learn how to do a “rebase.” And my skills haven't improved in other areas either! :)
I took the current data and merged it manually. I also made a few wording changes.

As always, I'm open to discussion.
After that, I would move on to 4.3.x (if necessary) and then 4.4.x to complete the 4.x series.

MCStreetguy commented:
Don't worry :) Git can be difficult to learn. It took me a long time to become even somewhat proficient, and that's only because I use it every day at work.

> I also made a few wording changes.

So most of the translations are still exactly the same as we discussed in the last PR? In that case I'll just skim through it again, since most of it should already be fine, or do you think I should read through it again in full?
Unfortunately I'm short on time today, but I think I'll definitely get to it tomorrow.

Also, @mhilbrunner, could you please check whether this PR can be merged without any problems? GitHub says there are still conflicts, but also that the changes could be cleanly merged. That sounds contradictory to me, but I don't have the permissions to look into it.

Szerina92 commented:
Most are the same, I basically used my old stuff, I just changed some keys using the official wording, since the phb is out now.
And I changed a few keys to streamline it a bit more.
I guess skimming is fine!

As soon as it's accepted I will start working on 4.3.x

I synced my fork and used the latest file, don't know why there are those conflicts :(

MCStreetguy commented:
Alright, glad to hear this, that saves me a lot of time :)

And don't worry too much about the conflicts, I'm quite sure this is only a misidentification if you just synced your fork. Which can happen occasionally. For some reason GitHub is not 100% sure it can merge the changes automatically, so it want's a human to have a look to avoid issues. As long as the changes can be merged without problems we can simply ignore that warning, but that is something the other Max has to take a look at since he's the only one with permissions to review the conflicts. In the unlikely case that there actually still are some, which I doubt, we'll also find a solution for that. :)

mhilbrunner commented:
Hey! Thanks again, both of you :) There are indeed some conflicts that prevent (auto) merging, I'll have a look.

We really need to get this thing up on Weblate to get away from the Git workflow just for translation. I'm setting aside some time this weekend to try to start that.

If you want to get in touch on Discord (I'm on the official Foundry Discord under the same name), maybe we should create a server there to discuss these things.

MCStreetguy commented:
I'm very sorry it took me so long, things have been very chaotic lately.
I have now had a look at all the changes and I have actually noticed the following two minor points:

- In `DND5E.CONSUMABLE -> Type -> Trinket -> Label`, "Schmuckstück" was renamed to "Requisite", meanwhile `DND5E.EquipmentTrinket` has been added as "Schmuckstück". It seems to me both these keys reference the same thing, shouldn't these be translated identically? Please correct me if I'm wrong.
- Under `DND5E.FLAGS.EnhancedDualWielding.Name` a previously corrected typo has been reintroduced ("Beidgändigkeit" instead of "Beidhändigkeit")

Also, for whatever reason, the following keys went back to their untranslated english form:

| Key | Previous | Now |
|:---:|:---|:---|
| `DND5E.AdvancementTraitModeUpgradeLabel` | "Verbesserung" | "Upgrade" |
| `DND5E.Award -> Message` | "{name} wurde {award} verliehen" | "{name} has been awarded {award}" |
| `DND5E.ConsumeScalingLabel` | "Ressourcen verwenden" | "Use Resources" |
| `DND5E.Flat` | "Flach" | "Flat" |
| `DND5E.FormulaMalformedError` | "Problem bei der Vorbereitung der Formel {property} in {name}." | "Problem preparing the {property} formula within {name}." |
| `DND5E.FormulaMissingReferenceWarn` | "Die {property}-Formel in {name} enthält Verweise auf fehlende Daten: {references}" | "The {property} formula within {name} has references to missing data: {references}" |
| `DND5E.Issues` | "Probleme" | "Issues" |
| `DND5E.TARGET -> FIELDS -> target -> template -> height -> hint` | "Höhe des betroffenen Zylinders, falls zutreffend." | "Height of a cylinder affected if applicable." |
| `DND5E.UseItem` | "{item} verwenden" | "Use {item}" |
| `DND5E.per` | "pro" | "per" |
| `SOURCE.BOOK.SRD` | "Systemreferenzdokument 5.1" | "System Reference Document 5.1" |

Other than that, well done! :)

Szerina92 commented:
Thanks for your time!

There are different "Trinkets".
I chose to change the Trinket key for Consumables because it's usually used for things like "Torch" and "Caltrops". Those never felt like Schmuckstücke for me. So I skimmed through some books where they translated a trinket table into german and they chose Requisite.
I'm fine if we change it back!
The other trinket is for equipment and feels right at least in some cases.

`Under DND5E.FLAGS.EnhancedDualWielding.Name a previously corrected typo has been reintroduced ("Beidgändigkeit" instead of "Beidhändigkeit")`

I guess there were some problems with syncing the fork, those are some weird errors!

MCStreetguy commented:
Okay, I see. Thank you for the explanation! In that case, I don't think we should change it back. I just looked at the translation keys, and they both mention "trinket," so I thought it might have been an oversight. However, you obviously thought this through and intentionally chose two different translations. Since this is even closer to the official German books, let's keep it that way. Sounds good to me! :)

Yeah, if I recall correctly, those were things I changed in my last 4.1 translation update. Especially the "Beidgändigkeit" error I clearly remember correcting. Most likely some error while syncing your fork. But whatever went wrong, at least those are pretty quick to rectify.

Freezor commented:
Is there anything I could do to help? I was waiting for a new release and did not want to create a new PR as there is the existing one

MCStreetguy commented:
@Freezor I don't think so, but thank you for the offer! Actually, everything had already been translated by @Szerina92; we just need to resolve the merge conflicts, and then a new release is all set to go.

mhilbrunner commented:
This module is being sunset, as discussed in https://github.com/mhilbrunner/foundryvtt-dnd5e-lang-de/issues/41#issuecomment-4296819410.

I suggest either hosting a fork with your changes or even better, chiming in on some of the alternative modules in development. Thanks for contributing :) Especially the 3 of you who have been pretty core to the effort for so long. Thank you, all of you!
